### PR TITLE
bump version to v0.5.1

### PR DESCRIPTION
**Pull request by lvhan028**

### CODE REVIEW

File: docs/en/supported_models/supported_models.md [hunk 1]
```diff
@@ -2,32 +2,33 @@
...
+|         QWen2         |  1.5B - 72B  |    Yes    |   Yes   |   Yes   |  Yes  |
+|        Mistral        |      7B      |    Yes    |   Yes   |   Yes   |  No   |
+|        QWen-VL        |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|      DeepSeek-VL      |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|       Baichuan        |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|       Baichuan2       |      7B      |    Yes    |   Yes   |   Yes   |  Yes  |
+|      Code Llama       |   7B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
+|          YI           |   6B - 34B   |    Yes    |   Yes   |   Yes   |  No   |
+|    LLaVA(1.5,1.6)     |   7B - 34B   |    Yes    |   Yes   |   Yes   |  Yes  |
+|     InternVL-Chat     |  v1.1- v1.5  |    Yes    |   Yes   |   Yes   |  Yes  |
```
Comment (zhyncs) [inline docs/en/supported_models/supported_models.md:27]: Shall we add InternVL 2 in the support table?

**Comment by irexyc** [comment]
I would recommend replacing all 0.5.0 with 0.5.1

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ ______________________________________________________________________
 <details open>
 <summary><b>2024</b></summary>
 
+- \[2024/07\] Support InternVL2 full-serie models, InternLM-XComposer2.5 and function call of InternLM2.5
```
Comment (irexyc) [inline README.md:29]: I would recommend add a link to the to xcomposer2d5 https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/multi_modal/xcomposer2d5.md

Comment (lvhan028) [reply README.md:29]: Sure

File: docs/en/supported_models/supported_models.md (hunk 1 above)
Comment (lvhan028) [reply docs/en/supported_models/supported_models.md:27]: Yes. Added.

File: docs/en/supported_models/supported_models.md [hunk 3]
```diff
@@ -2,32 +2,34 @@
...
+|       InternVL2       |    1B-40B    |    Yes    |   Yes   |   Yes   |  Yes  |
```
Comment (zhyncs) [inline docs/en/supported_models/supported_models.md:28]: InternVL2 1B is not compatible with TurboMind but is supported by the PyTorch Engine. Perhaps we may clarify this.

Comment (zhyncs) [reply docs/en/supported_models/supported_models.md:28]: It has two tables.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by zhyncs — APPROVED** [review]

**Pull request merged**